src/hooks/api/collections/useCreateCollection.ts

```diff
@@ -78,6 +79,7 @@ export default function useCreateCollection() {
             tokens: tokenIds,
             layout,
             tokenSettings: tokenSettingsArray,
+            ...(caption && { caption }),
```

Comment:
does `caption: caption ?? null` work? this syntax is a bit wonky for me

Reply:
or ?? undefined or whatever gets the job done

Reply:
I didn't remember why I came into this solution 🤦 . Updated it with `aption: caption ?? null`